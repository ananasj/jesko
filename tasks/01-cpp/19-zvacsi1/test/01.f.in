36
22
38 22 93 46 3 76 96 58 94 54 49 34 32 56 80 70 50 98 84 14 26 43 31 20 64 74 72 44 95 96 94 79 53 66 44 38
