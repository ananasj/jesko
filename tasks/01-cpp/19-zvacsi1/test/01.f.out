60 44 115 68 25 98 118 80 116 76 71 56 54 78 102 92 72 120 106 36 48 65 53 42 86 96 94 66 117 118 116 101 75 88 66 60
